Docs/source/developers/how_to_test.rst

```diff
@@ -230,41 +209,3 @@ Note that we currently obey the following snake\_case naming conventions for tes
 #. **Test input files** start with the string ``inputs_`` followed by the test name. For example, ``inputs_test_3d_laser_acceleration`` or ``inputs_test_3d_laser_acceleration_picmi.py`` or ``inputs_test_3d_laser_acceleration_restart``.
 
 #. **Base input files** (that is, files collecting input parameters shared between two or more tests) are typically named ``inputs_base_1d``, ``inputs_base_2d``, ``inputs_base_3d`` or ``inputs_base_rz``, possibly followed by additional strings if need be.
-
-Useful tool for plotfile comparison: ``fcompare``
--------------------------------------------------
-
-AMReX provides ``fcompare``, an executable that takes two ``plotfiles`` as input and returns the absolute and relative difference for each field between these two plotfiles. For some changes in the code, it is very convenient to run the same input file with an old and your current version, and ``fcompare`` the plotfiles at the same iteration. To use it:
-
-.. code-block:: sh
-
-   # Compile the executable
-   cd <path to AMReX>/Tools/Plotfile/ # This may change
-   make -j 8
-   # Run the executable to compare old and new versions
-   <path to AMReX>/Tools/Plotfile/fcompare.gnu.ex old/plt00200 new/plt00200
-
-which should return something like
-
-.. code-block:: sh
-
-             variable name             absolute error            relative error
-                                          (||A - B||)         (||A - B||/||A||)
-   ----------------------------------------------------------------------------
-   level = 0
-   jx                                 1.044455105e+11               1.021651316
-   jy                                  4.08631977e+16               7.734299273
-   jz                                 1.877301764e+14               1.073458933
-   Ex                                 4.196315448e+10               1.253551615
-   Ey                                 3.330698083e+12               6.436470137
-   Ez                                 2.598167798e+10              0.6804387128
-   Bx                                     273.8687473               2.340209782
-   By                                     152.3911863                1.10952567
-   Bz                                     37.43212767                 2.1977289
-   part_per_cell                                   15                    0.9375
-   Ex_fp                              4.196315448e+10               1.253551615
-   Ey_fp                              3.330698083e+12               6.436470137
-   Ez_fp                              2.598167798e+10              0.6804387128
-   Bx_fp                                  273.8687473               2.340209782
-   By_fp                                  152.3911863                1.10952567
-   Bz_fp                                  37.43212767                 2.1977289
```

Comment:
Do developers really use this when adding automated tests? If it's a tool that isn't used much, I would rather remove this section to keep the documentation as concise as possible.

Reply:
I do use fcompare occasionally. But I agree that this level of description is not needed. Maybe some place here just mention that that tool is available.